seeders/20170627154705-example-seeds.js

```diff
@@ -0,0 +1,70 @@
+const uuid = require('uuid');
+const moment = require('moment');
+
+const authorOneId = uuid.v4();
+const authorTwoId = uuid.v4();
+const authorThreeId = uuid.v4();
+
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.bulkInsert('Authors', [
+      {
+        id: authorOneId,
+        firstName: 'David',
+        lastName: 'Van Reybrouck',
+        createdAt: moment().toISOString(),
+        updatedAt: moment().toISOString(),
+      },
+      {
+        id: authorTwoId,
+        firstName: 'Robert',
+        lastName: 'Martin',
+        createdAt: moment().toISOString(),
```

Comment:
When defining `timestamps: true` on a sequelize model, the `updatedAt` & `createdAt` values are automatically updated. More over here: http://docs.sequelizejs.com/manual/tutorial/models-definition.html#configuration